integrations/examples/calling-guest-functions.md

```diff
@@ -113,6 +128,10 @@ result, err := sum.Call(1, 2)
 ```
 {% endtab %}
 
+{% tab title="Python" %}
+(not possible)
```

Comment:
Can we either explain why it's not possible or simply remove this?

Having this message formatted in a nice `{% hint style="warn" %}` would be great. Having justification about why it's not possible and something saying to look at the native flavor would be perfect I think.